frontend/src/chat.css

```
@@ -625,6 +664,96 @@
+  background-color: transparent;
+  box-shadow: none;
+  border: none;
+  border-radius: 0;
+}
+
+.markdown-body table.markdown-table-maple thead th,
+.markdown-body table.markdown-table-maple tbody td,
+.markdown-body table.markdown-table-maple tbody th {
+  min-width: 0;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)